openfecwebapp/templates/partials/search-results-alternatives.html

```diff
@@ -1,11 +1,11 @@
 <div class="message message--alert">
   {% if not results %}
   <h2 class="message__title">No results</h2>
-  <p>Sorry, no results match your search, <span class="t-bold">"{{ query }}."</span></p>
+  <p>Sorry, no results match your search, <span class="t-bold">“{{ query }}.”</span></p>
```

Comment:
I think this should say `<p>Sorry, no results match your search for <span class="t-bold">“{{ query }}.”</span></p>`